### ISSUE TITLE
[BUG] hud modules not working, new folder not created?

### ISSUE THREAD

**Issue by modzy2** [Larkinabout/fvtt-token-action-hud-core#114]
**Describe the bug**

hud core seems to have created the new directory in Data, but no directory of the world name under that.
Not sure if the backslash instead of forward slash is involved

token-action-hud-core.min.js:1 Uncaught (in promise) ENOENT: no such file or directory, mkdir '/media/pi/HIRESAUDIO1/FDATA10/Data/token-action-hud-core\stormwreck'
(anonymous)	@	token-action-hud-core.min.js:1
await in (anonymous) (async)		
#call	@	foundry.js:724
call	@	foundry.js:706
(anonymous)	@	token-action-hud-dnd5e.min.js:1
#call	@	foundry.js:724
callAll	@	foundry.js:681
(anonymous)	@	token-action-hud-core.min.js:1
#call	@	foundry.js:724
callAll	@	foundry.js:681
setupGame	@	foundry.js:7952
await in setupGame (async)		
_initializeGameView	@	foundry.js:8999
_initializeView	@	foundry.js:8975
initialize	@	foundry.js:7825
await in initialize (async)		
🎁call_wrapped	@	libWrapper-wrapper.js:507
🎁libWrapperInit	@	libWrapper-api.js:805
await in 🎁libWrapperInit (async)		
🎁Game.prototype.initialize#0	@	libWrapper-wrapper.js:187
window.addEventListener.once	@	foundry.js:79693

**Steps to reproduce**

**Versions:**
- Token Action HUD Core: [e.g. 0 1.0]
- Token Action HUD System: [e.g. token-action-hud-dnd5e 0.1.0]
- System: [e.g. dnd5e 2.0.3]
- Foundry VTT: [e.g. 10.291]
- Browser: [e.g. Google Chrome 108.0.5359.124]

**Additional context**
Any additional context.

**Comment by farling42** [comment]
This was caused by the socketlib module not being enabled - but strangely no error was reported telling me that socketlib needed to be enabled.

**Comment by Larkinabout** [comment]
Well, that's a weird one as it creates the directories okay on my side. It should check and try to create the directories everytime you reload the world.  Does it throw that error everytime?

**Comment by modzy2** [comment]
yes I logged off and on a few times, still getting that error and nothing created inside the token-action-hud-core folder in "data"

**Comment by Larkinabout** [comment]
Would you be able to create a new world and see if that one has the same issue?

**Comment by modzy2** [comment]
yes, same error.

**Comment by Larkinabout** [comment]
What hardware/OS are you hosting on?

**Comment by Larkinabout** [comment]
Looks like Windows accepts either case, so I'll swap them around. Hopefully, I don't have to try and determine which case to use based on the OS.

**Comment by MarcoDaddix** [comment]
Don't worry I'm online right now, I can confirm in about minutes :)

**Comment by Larkinabout** [comment]
Hopefully fixed in Token Action HUD Core 1.4.2.

**Comment by MarcoDaddix** [comment]
Confirmed: it works now. Directories are created correctly now. Thanks :)
